Libplanet.Tests/Menees.Analyzers.Settings.xml

```diff
@@ -2,5 +2,5 @@
 <Menees.Analyzers.Settings>
   <MaxLineColumns>100</MaxLineColumns>
   <MaxMethodLines>300</MaxMethodLines>
-  <MaxFileLines>2600</MaxFileLines>
+  <MaxFileLines>2700</MaxFileLines>
```

Comment:
Rather than increasing this, how about making a partial class of `SwarmTest` instead?